dna/tests/src/project/project_init.rs

```diff
@@ -0,0 +1,84 @@
+#[cfg(test)]
+pub mod tests {
+    use crate::project::fixtures::fixtures::ProjectMetaFixturator;
+    use ::fixt::prelude::*;
+    use hdk::prelude::*;
+    use hdk_crud::WrappedAgentPubKey;
+    use holochain_types::prelude::option_entry_hashed;
+    use holochain_types::prelude::ElementFixturator;
+    use holochain_types::prelude::ValidateDataFixturator;
+    use projects::project::error::Error;
+    use projects::project::project_meta::crud::ProjectMeta;
+    use projects::project::project_meta::validate::*;
+    use projects::init;
+
+    #[test]
+    fn test_init() {
+               
```

Comment:
let's add a comment here that describes the overall behaviour of `init` that we're testing, i.e. the fact that it makes two main function calls, and most importantly the purpose of those function calls, focused on being human readable